src/urllib3/response.py

```diff
@@ -720,6 +781,48 @@ def _fp_read(self, amt: Optional[int] = None) -> bytes:
             # StringIO doesn't like amt=None
             return self._fp.read(amt) if amt is not None else self._fp.read()
 
+    def _raw_read(
+        self,
+        amt: Optional[int] = None,
+    ) -> bytes:
+        """
+        Reads `amt` of bytes from the socket.
+        """
+        if self._fp is None:
+            return None  # type: ignore[return-value]
```

Comment:
Should this return `b""`? Otherwise we should type annotate this function properly.

Reply:
This pull request does not change anything here, we should address this independently imo.